src/probability/kernel/composition.lean

```diff
@@ -0,0 +1,567 @@
+/-
+Copyright (c) 2022 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+
+import probability.kernel.basic
+
+/-!
+# Product and composition of kernels
+
+Given s-finite kernels `κ : kernel α β` and `η : kernel (α × β) γ`, we define their product
```

Comment:
```suggestion
Given `s`-finite kernels `κ : kernel α β` and `η : kernel (α × β) γ`, we define their product
```
To indicate that `s` is a variable

Reply:
s is not a variable here. The property of being equal to a countable sum of finite kernels is called s-finite.

Reply:
Apologies, I got tricked by the `s` that appears below!